src/azure-cli-core/azure/cli/core/cloud.py

```diff
@@ -267,7 +267,11 @@ def get_clouds(cli_ctx):
     # Start off with known clouds and apply config file on top of current config
     for c in KNOWN_CLOUDS:
         _config_add_cloud(config, c)
-    config.read(CLOUD_CONFIG_FILE)
+    try:
+        config.read(CLOUD_CONFIG_FILE)
+    except configparser.MissingSectionHeaderError:
+        os.remove(CLOUD_CONFIG_FILE)
+        logger.debug("'%s' is in bad format and has been removed.", CLOUD_CONFIG_FILE)
```

Comment:
I suggest `logger.warning` as this is a file containing settings visible to users